tests/test_meshtastic_utils.py

```
@@ -2260,5 +2260,552 @@ def test_check_connection_uncovered_paths(self, mock_logger):
+        mock_wait_for,
+        mock_get_running_loop,
+    ):
+        """Test _disconnect_ble_by_address handles unexpected disconnect errors."""
+        from mmrelay.meshtastic_utils import _disconnect_ble_by_address
+
+        async def _noop(*_args, **_kwargs):
+            return None
+
+        mock_get_running_loop.side_effect = RuntimeError("no loop")
+        mock_sleep.side_effect = _noop
+        mock_wait_for.side_effect = lambda awaitable, timeout=None: awaitable
```

Ruff (0.14.13)

Unused lambda argument: `timeout`

(ARG005)